testing/install/CMakeLists.txt

```diff
@@ -17,9 +17,6 @@ function(add_install_cmake_test name)
     "-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}"
     "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
     "-DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}"
-    "-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}"
```

Comment:
@chuckatkins This was needed to make this test work. Note that the only flags that were being passed was -Wall -Werror. It seems that we did not need this.

Reply:
We need to keep this in.  If a user specifies certain compilation flags then leaving this out can cause the adios-config script generated at install time to produce incorrect flags.

Reply:
I see, right! Find this resolved at the last push